scripts/create_glutton_spec.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# Example usage to:
+#
+# - Use the `polkadot-parachain` binary;
+# - Use `rococo` as the parent Relay Chain;
+# - Generate `ParaId`s from 1,300 to 1,370, inclusive;
+# - Set the Sudo key to `G7Z5mTmTQsjEGBVqVGDZyR9m7RoHNZJk6JeykyfKQ3vmBiR`;
+# - Set `compute`, `storage`, and `trash_data_count` set to 50%, 50%, and 5,120, respectively;
+# - And save the results in `output-dir`.
+#
+# ./scripts/create_glutton_spec.sh ./target/release/polkadot-parachain rococo 1300 1370 G7Z5mTmTQsjEGBVqVGDZyR9m7RoHNZJk6JeykyfKQ3vmBiR 500000000 500000000 5120 output-dir
+
+usage() {
+    echo Usage:
+    echo "$0 <binary path> <relay chain> <from parachain id> <to parachain id> <sudo key> <compute> <storage> <trash_data_count> <output dir>"
+    exit 1
+}
+
+set -e
+
+binary_path=$1
+relay_chain=$2
+from_para_id=$3
+to_para_id=$4
+sudo=$5
+compute=$6
+storage=$7
+trash_data_count=$8
+output_dir=$9
+
+[ -z "$binary_path" ] && usage
+[ -z "$relay_chain" ] && usage
+[ -z "$from_para_id" ] && usage
+[ -z "$to_para_id" ] && usage
+[ -z "$sudo" ] && usage
+[ -z "$compute" ] && usage
+[ -z "$storage" ] && usage
+[ -z "$trash_data_count" ] && usage
+[ -z "$output_dir" ] && usage
+
+
+for (( para_id=$from_para_id; para_id<=$to_para_id; para_id++ )); do
+    echo "Building chain specs for parachain $para_id"
+
+    # create dir to store parachain generated files
+    output_para_dir="$output_dir/glutton-$relay_chain-$para_id"
+    if [ ! -d "$output_para_dir" ]; then
+        mkdir $output_para_dir
+    fi
+
+    # build the chain spec we'll manipulate
+    $binary_path build-spec --disable-default-bootnode --chain "glutton-kusama-genesis-$para_id" > "$output_para_dir/plain-glutton-$relay_chain-$para_id-spec.json"
+
+    id="glutton-$relay_chain-$para_id"
+    protocol_id="glutton-$relay_chain-$para_id"
+
+    # replace the runtime in the spec with the given runtime and set some values to production
+    cat "$output_para_dir/plain-glutton-$relay_chain-$para_id-spec.json" \
+        | jq --arg id $id '.id = $id' \
```

Comment:
We should ensure `jq` is installed at the beginning, otherwise abort the execution and instruct to install it.